src/config.h.in

```diff
@@ -20,7 +20,7 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-#cmakedefine CMAKE_PROJECT_VERSION "${CMAKE_PROJECT_VERSION}"
+#cmakedefine WABT_VERSION_STRING "@WABT_VERSION_STRING@"
```

Comment:
What is the difference between the $ syntax and the @ syntax?

Reply:
Nothing functional, but the rest of the file favors the `@` syntax.

Reply:
I see.. cmake supports two different syntaxs and the @ONLY limits it to just the @ version.. sgtm.